File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/UserPermissionId.java
package org.dddml.wms.domain;


public class UserPermissionId
{
    private String userId;

    public String getUserId()
    {
        return this.userId;
    }

    public void setUserId(String userId)
    {
        this.userId = userId;
    }

    private String permissionId;

    public String getPermissionId()
    {
        return this.permissionId;
    }

    public void setPermissionId(String permissionId)
    {
        this.permissionId = permissionId;
    }

    UserPermissionId ()
    {
    }

    public UserPermissionId (String userId, String permissionId)
    {
        this.userId = userId;
        this.permissionId = permissionId;
    }

}

File: Dddml.Wms.AdminUI/ts/UserPermissionId.ts
namespace Dddml.Wms.Domain {

	export class UserPermissionId {

		private userId: string;

		getUserId(): string {
			return this.userId;
		}

		setUserId(userId: string) {
			this.userId = userId;
		}

		private permissionId: string;

		getPermissionId(): string {
			return this.permissionId;
		}

		setPermissionId(permissionId: string) {
			this.permissionId = permissionId;
		}

	}
}

